src/formatDate.js

```diff
@@ -50,7 +50,64 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const DATE_PARTS = date.split(fromFormat[fromFormat.length - 1]);
+  let newDate = '';
+
+  if (toFormat[0] === 'YYYY') {
```

Comment:
I need check on line 56, because I have a problem with test where 'DD' should come first. And this check helps to solve this problem. And these checks use loops where 'const INDEX' is different, so there must be two conditions